src/Tizen.Multimedia.MediaPlayer/Player/AudioOffload.cs

```diff
@@ -0,0 +1,188 @@
+        /// This will reduce the power consumption, but will disable the ability to handle output PCM.
+        /// Please check the below list of functions which will not work if offloading is enabled.</para>
+        /// <para>If audio offload is enabled, the following functions will return <see cref="InvalidOperationException"/>
+        /// and they will not work at all even if they were called before offload is enabled.<br/>
+        /// <see cref="AudioEffect"/><br/>
+        /// <see cref="EqualizerBand"/><br/>
+        /// <see cref="PlayerTrackInfo"/><br/>
+        /// <see cref="Player.EnableExportingAudioData"/><br/>
+        /// <see cref="Player.AudioLatencyMode"/><br/>
+        /// <see cref="Player.SetPlaybackRate"/><br/>
+        /// <see cref="Player.ReplayGain"/><br/>
+        /// <see cref="Player.AudioPitch"/><br/>
+        /// <see cref="Player.AudioPitchEnabled"/><br/></para>
+        /// <para>To set, the player must be in the <see cref="PlayerState.Idle"/> state.
+        /// The sound stream type of the player should be <see cref="AudioStreamType.Media"/>.</para></remarks>
+        /// <feature>http://tizen.org/feature/multimedia.player.audio_offload</feature>
+        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">
+        ///     The player is not in the valid state.<br/>
+        ///     -or-<br/>
+        ///     Operation failed; internal error.
+        /// </exception>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsEnabled
+        {
+            get
+            {
+                ValidationUtil.ValidateFeatureSupported(PlayerFeatures.AudioOffload);
+                _player.ValidateNotDisposed();
+
+                NativePlayer.IsAudioOffloadEnabled(_player.Handle, out var value).
+                    ThrowIfFailed(_player, "Failed to get whether the audio offload of the player is enabled or not");
+                return value;
+            }
+
+            set
+            {
+                ValidationUtil.ValidateFeatureSupported(PlayerFeatures.AudioOffload);
+                _player.ValidateNotDisposed();
+                _player.ValidatePlayerState(PlayerState.Idle);
+
+                NativePlayer.SetAudioOffloadEnabled(_player.Handle, value).
+                    ThrowIfFailed(_player, "Failed to set the audio offload of the player");
+                _enabled = value;
+            }
+        }
+
+        /// <summary>
+        /// Get a state whether or not the audio offload is activated.
+        /// </summary>
+        /// <value>The value indicating whether or not AudioOffload is activated.</value>
+        /// <remarks>
+        /// Audio offload could be inactivated depending on the audio device capability even if the audio offload feature is supported.
```

Comment:
Is "even if" correct in this sentence?
Reference:  https://www.englishgrammar.org/even-if-and-even-though/

What about the following sentence?
        /// It depends on the audio device capability whether audio offload can activated even though the audio offload feature is supported.